renderdoc/driver/vulkan/vk_core.h

```diff
@@ -377,6 +378,8 @@ class WrappedVulkan : public IFrameCapturer
     VkFormatProperties fmtprops[VK_FORMAT_RANGE_SIZE] = {};
     VkDriverInfo driverInfo = VkDriverInfo(props);
 
+    VkPhysicalDevicePerformanceQueryFeaturesKHR performance_query_features = {};
```

Comment:
I'm not terribly consistent with naming schemes but please try to match the surrounding code - the members here are mostly camelCase so that would be better for this variable.

Reply:
Sure.